README.md

```diff
@@ -1,141 +1,161 @@
-* Install tomcat7: `apt-get install tomcat7`.
-* Install mysql: `apt-get install mysql-server`.
-* Login to mysql: `mysql -u [user] -p`.
-* Create db:`create dababase scadalts;`.
-* Copy ScadaBR.war & ScadaLTS.war to dir tomacat webapps: `cp ScadaBR.war /var/lib/tomcat7/webapps/`.
-* Restart tomcat7 to generate ScadaLTS folder: `/etc/init.d/tomcat7 restart`
-* Change config file env.properties set connection to db: `vim /var/lib/tomcat7/webapps/ScadaBR/WEB-INF/classes/env.properties`.
-* Restart tomcat7: `/etc/init.d/tomcat7 restart`
-
-
-### What's included
-
-App ScadaBR.war 
-App ScadaLTS.war
-
-## Bugs and feature requests
-
-Have a bug or a feature request? Please first read the [issue guidelines](https://github.com/twbs/bootstrap/blob/master/CONTRIBUTING.md#using-the-issue-tracker) and search for existing and closed issues. If your problem or idea is not addressed yet, [please open a new issue](https://github.com/sdtabilit/Scada-LTS/issues/new).
-
-## Documentation
-
-Scada-LTS documentation, will be included in this repo in the root directory.
-
-
-## Contributing
-
-In the process of making.
-
-## Community
-
-* ScadaLTS tag is finally available on Stack Overflow. Feel free to ask questions - http://stackoverflow.com/questions/tagged/scadalts.
-
-## Tests
-
-[scada-lts.testquality.com](https://scada-lts.testquality.com)
-
-
-## Versioning
-
-For transparency into our release cycle and in striving to maintain backward compatibility, Scada-LTS is maintained under [the Semantic Versioning guidelines](http://semver.org/). Sometimes we screw up, but we'll adhere to those rules whenever possible.
-
-See [the Releases section of our GitHub project](https://github.com/grzesiekb/Scada-LTS/releases) for changelogs for each release version of Scada-LTS. 
-
-## Creators 
-
-Code base on [ScadaBR](https://sourceforge.net/projects/scadabr/?source=directory)
-
-**Michał Rokitiański**
-
-* <https://github.com/SoftQ>
-
-**Grzesiek Bylica**
-
-* <https://github.com/grzesiekb>
-
-**Arkadiusz Parafiniuk**
-
-* <https://github.com/ArkadiuszParafiniuk>
-
-**Konrad Daniek**
-
-* <https://github.com/wisyr>
-
-**Jarosław Morzyniec**
-
-* <https://github.com/Morzyniec> Testing
-
-**Mateusz Kaproń**
-
-* <https://github.com/matkapron>
-
-**Diego Rodrigues Ferreira**
-
-* <https://github.com/katesclau>
-
-**Jerzy Piejko**
-
-* <https://github.com/JerzyPiejko>
-
-**Radosław Jajko**
-
-* <https://github.com/radek2s>
-
-**Marcin Gołda**
-
-* <https://github.com/marcingolda>
-
-**Tomasz Skrobol**
-
-* <https://github.com/TomashHawk>
-
-## Support
-
-If you need support for deployment please contact us and find more information on [our website](http://scada-lts.comhttp://scada-lts.com/#support)
-
-
-## Copyright and license
-
-Code released under [the GPL license](https://github.com/sdtabilit/Scada-LTS/blob/master-sdtabilit/LICENSE). 
+# [Scada-LTS](http://scada-lts.org)
```

Comment:
You probably changed the encoding of the file README.md than We can't see what has been changed in README.md.